src/python/test/test_dtm.py

```diff
@@ -91,13 +91,11 @@ def test_density():
 
 def test_dtm_overflow_warnings():
     pts = numpy.array([[10., 100000000000000000000000000000.], [1000., 100000000000000000000000000.]])
-    impl_warn = ["keops", "hnsw"]
 
     with warnings.catch_warnings(record=True) as w:
-        for impl in impl_warn:
-            dtm = DistanceToMeasure(2, q=10000, implementation=impl)
-            r = dtm.fit_transform(pts)
-        assert len(w) == 2
-        for i in range(len(w)):
-            assert issubclass(w[i].category, RuntimeWarning)
-            assert "Overflow" in str(w[i].message)
+        # TODO Test "keops" implementation as well when next version of pykeops (current is 1.5) is released (should fix the problem (cf. issue #543))
+        dtm = DistanceToMeasure(2, q=10000, implementation="hnsw")
```

Comment:
While we are there, as noted in the issue, this `q=10000` is strange, it is not passed to knn but we expect the warning to come from knn. Should we skip it?

Reply:
Yes, indeed !